tensorflow/lite/micro/kernels/fully_connected_test.cc

```diff
@@ -620,4 +632,38 @@ TF_LITE_MICRO_TEST(SimpleTestQuantizedInt8NullBias) {
       kTfLiteOk);
 }
 
+// INT4 isn't supported on Hexagon (b/258710417)
```

Comment:
let's make this `TODO(b/258710417): INT4 isn't currently supported on Hexagon.` That way the bug will have a link to all the TODO references in the code.

Reply:
Fixed.